lib/open_food_network/sales_tax_report.rb

```diff
@@ -9,9 +9,27 @@ def initialize(user, params)
     end
 
     def header
-      ["Order number", "Date", "Items", "Items total (#{currency_symbol})", "Taxable Items Total (#{currency_symbol})",
-        "Sales Tax (#{currency_symbol})", "Delivery Charge (#{currency_symbol})", "Tax on Delivery (#{currency_symbol})", "Tax on Fees (#{currency_symbol})",
-        "Total Tax (#{currency_symbol})", "Customer", "Distributor"]
+      case params[:report_type]
+      when "tax_types"
+        [I18n.t(:report_header_order_number),
+         I18n.t(:report_header_date),
+         I18n.t(:report_header_items),
+         I18n.t(:report_header_items_total, currency_symbol: currency_symbol),
+         I18n.t(:report_header_taxable_items_total, currency_symbol: currency_symbol),
+         I18n.t(:report_header_sales_tax, currency_symbol: currency_symbol),
+         I18n.t(:report_header_delivery_charge, currency_symbol: currency_symbol),
+         I18n.t(:report_header_tax_on_delivery, currency_symbol: currency_symbol),
+         I18n.t(:report_header_tax_on_fees, currency_symbol: currency_symbol),
+         I18n.t(:report_header_total_tax, currency_symbol: currency_symbol),
+         I18n.t(:report_header_customer),
+         I18n.t(:report_header_distributor)]
+      else
+        [I18n.t(:report_header_order_number),
+         I18n.t(:report_header_total_excl_vat, currency_symbol: currency_symbol)] +
+        relevant_rates.map { |rate| "%s (%.1f%%) (%s)" % [rate.name, rate.amount.to_f * 100, currency_symbol] } +
```

Comment:
Just thinking that `relevant_rates` will return a copy of both the tax-inclusive and tax-exclusive versions as separate rates if they both exist, and are both used. This may be a fairly rare case, but just thought I would raise it. @myriamboure, what do you think? Should we be reporting taxes accumulated under tax-inclusive and tax-exclusive versions of the same rate in the same column?